cmd/fixer.go

```diff
@@ -41,6 +41,16 @@ var softwareIDs = map[string]string{
 var fixerCmdGroup = &cobra.Command{
 	Use:   "fixer <command>",
 	Short: "A set of tools to fix issues or migrate content for retro-compatibility.",
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		if err := config.Setup(cfgFile); err != nil {
+			return err
+		}
+		if config.FsURL().Scheme != config.SchemeSwift &&
+			config.FsURL().Scheme != config.SchemeSwiftSecure {
+			return fmt.Errorf("swift: the configured filesystem does not rely on OpenStack Swift")
```

Comment:
Hey, it looks like it make it impossible to run the fixer command for the self-hosted and dev docker images!